SwiftCompilerSources/CMakeLists.txt

```diff
@@ -106,6 +106,13 @@ function(add_swift_compiler_modules_library name)
       "-Xcc" "-std=c++17"
       "-Xcc" "-DCOMPILED_WITH_SWIFT" "-Xcc" "-DSWIFT_TARGET"
       "-Xcc" "-UIBOutlet" "-Xcc" "-UIBAction" "-Xcc" "-UIBInspectable")
+  
+  if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    list(APPEND swift_compile_options "-static")
+    # Workaround a crash in the LoadableByAddress pass
```

Comment:
Is there a radar and Swift issue that tracks this?

Reply:
Yes, I just created github issues and added the links in comments